src/base/bittorrent/session.cpp

```diff
@@ -1444,12 +1444,16 @@ void Session::configureNetworkInterfaces(lt::settings_pack &settingsPack)
         const QHostAddress addr {ip};
         if (!addr.isNull())
         {
-            const QString ip = ((addr.protocol() == QAbstractSocket::IPv6Protocol)
-                          ? ('[' + Utils::Net::canonicalIPv6Addr(addr).toString() + ']')
+            const bool isIPv6 = addr.protocol() == QAbstractSocket::IPv6Protocol;
+            const QString ip = (isIPv6
+                          ? (Utils::Net::canonicalIPv6Addr(addr).toString())
                           : addr.toString());
-            endpoints << (ip + portString);
 
-            if ((ip != "0.0.0.0") && (ip != "[::]"))
+            endpoints << (isIPv6
+                          ? QString('[' + ip + ']') + portString
+                          : ip + portString);
+
+            if ((ip != "0.0.0.0") && (ip != "::"))
```

Comment:
Some clarification:
We don't need `QLatin1String` here. The test is ok the way it is.

Reply:
>We don't need QLatin1String here. The test is ok the way it is.

QLatin1String will make it a bit faster since it avoids converting from UTF-8.

Reply:
All right. Wasn't like this before but no objection from my side.